assets/components/src/text-control/style.scss

```
@@ -11,12 +11,22 @@
 	line-height: 24px;
 	margin: 32px 0;
 
+
 	label {
 		color: $dark-gray-900;
 		font-size: inherit;
 		font-weight: bold;
 		line-height: inherit;
 		margin: 0;
+
+		&[data-required-text] {
+			&::after {
+				content: attr( data-required-text );
+				padding-left: 5px;
+				font-weight: normal;
+				opacity: 0.5;
```

Minor nitpick, but suggest swapping the opacity here for one of the existing mid-greys used in the UI, like:

```suggestion
				 color: $dark-gray-300;
```

---

Done in 5ae9711750d7e43a80fd15b32bcfed700c7e849b

---

Thanks!